examples/topoflow4/topoflow_climate.yml

```diff
@@ -4,10 +4,11 @@ adapters:
     comment: My topoflow climate write adapter
     adapter: funcs.Topoflow4ClimateWriteFunc
     inputs:
-      input_dir: "/tmp/demo/input/weather"
-      temp_dir: "/tmp/demo/tmp"
-      output_file: "/tmp/demo/output/climate_all.zip"
-      var_name: precipitation
-      DEM_bounds: "23.995416666666, 6.532916666667, 28.020416666666, 9.566250000000"
-      DEM_xres_arcsecs: "30"
-      DEM_yres_arcsecs: "30"
+      input_dir: "/home/rook/workspace/mint/MINT-Transformation/data/GLDAS"
```

Comment:
This one is currently used as example for MITRE test. All of the paths should be changed